cluster/saltbase/salt/etcd/etcd.manifest

```diff
@@ -27,7 +27,7 @@
 "containers":[
     {
     "name": "etcd-container",
-    "image": "gcr.io/google_containers/etcd:3.0.3",
+    "image": "gcr.io/groovy-sentry-504/etcd:3.0.4",
```

Comment:
@lavalamp - this is the result of calling "make push" on etcd images directory after changes from this PR.